mule-packager/src/main/java/org/mule/tools/api/packager/builder/PackageBuilderFactory.java

```diff
@@ -6,15 +6,18 @@
  */
 package org.mule.tools.api.packager.builder;
 
+import org.mule.tools.api.packager.packaging.PackagingOptions;
+
 /**
  * Factory for package builders.
  */
 public class PackageBuilderFactory {
 
   /**
    * @return
```

Comment:
could we put a simple comment here 'returns a {@link PackageBuilder} based on the packaging options'